spyder/utils/site/sitecustomize.py

```diff
@@ -423,12 +423,15 @@ def user_return(self, frame, return_value):
 
 @monkeypatch_method(pdb.Pdb, 'Pdb')
 def interaction(self, frame, traceback):
-    self.setup(frame, traceback)
-    if self.send_initial_notification:
-        self.notify_spyder(frame)
-    self.print_stack_entry(self.stack[self.curindex])
-    self._cmdloop()
-    self.forget()
+    if "sitecustomize.py" not in frame.f_code.co_filename:

```

Comment:
Please verify that this works in Python 2

Reply:
Maybe this needs `u'sitecustomize.py'`

Reply:
Yep it works :+1: